### PR TITLE
test: run-standalone.sh: point LD_LIBRARY_PATH to $(pwd)/lib

### PR DESCRIPTION
**Pull request by dzafman**
- [x] References tracker ticket

### CODE REVIEW

File: install-deps.sh [hunk 1]
```diff
@@ -433,7 +433,11 @@ find . -name tox.ini | while read ini ; do
         fi
     )
 done
+curl -O https://bootstrap.pypa.io/get-pip.py
+$(SUDO) python get-pip.py
+$(SUDO) pip install pyopenssl
```
Comment (tchaikov) [inline install-deps.sh]: python-openssl is installed as a dependency of ceph-mgr. so presumably it should be around after running `install-deps.sh`. could you pastebin the error message if pyopenssl is not installed with `pip`?

File: qa/run-standalone.sh [hunk 2]
```diff
@@ -55,6 +55,9 @@ PATH=$(pwd)/bin:$PATH
 # user's PATH does not get these directories by default (e.g., tumbleweed)
 PATH=$PATH:/sbin:/usr/sbin
 
+export LD_LIBRARY_PATH="$(pwd)/lib"
+sudo ldconfig
```
Comment (tchaikov) [inline qa/run-standalone.sh]: no need to run `ldconfig`, as we are not changing `/etc/ld.so.conf` here.

File: install-deps.sh (hunk 1 above)
Comment (dzafman) [reply install-deps.sh]: HEALTH_WARN Module 'restful' has failed dependency: No module named OpenSSL
MGR_MODULE_DEPENDENCY Module 'restful' has failed dependency: No module named OpenSSL
    Module 'restful' has failed dependency: No module named OpenSSL

File: qa/run-standalone.sh (hunk 2 above)
Comment (dzafman) [reply qa/run-standalone.sh]: I'll fix this.

File: install-deps.sh (hunk 1 above)
Comment (tchaikov) [reply install-deps.sh]: ah! i see. `python-openssl` is listed as a runtime dependency but not a *build*-time dependency. that's why it is not installed by `install-deps.sh`! see @tserong 's analysis at https://tracker.ceph.com/issues/37584#note-13 .

so this PR is actually fixing https://tracker.ceph.com/issues/37584 . i am proposing an alternative to address this issue at https://tracker.ceph.com/issues/37584#note-15. and a fix is posted at #26402

**Comment by dzafman** [comment]
jenkins test make check

**Comment by dzafman** [comment]
This was testing in my build tree as that is where run-standalone.sh is done.

**Pull request merged**